backend/main.py

```diff
@@ -97,11 +91,73 @@ def solve():
     else:
         abort(403)
 
-@app.route('/register', methods=['POST'])
+@app.route('/login', methods = ['POST'])
+def login():
+    email = request.form.get('email')
+    attempt = request.form.get('password')
+
+    db_hostname = os.environ['COS_DB_HOST']
+    db_username = os.environ['COS_DB_USERNAME']
+    db_password = os.environ['COS_DB_PASSWORD']
+    db_name = os.environ['COS_DB_DATABASE']
+    conn = psycopg2.connect(host=db_hostname, user=db_username,
+                            password=db_password, dbname=db_name)
+    
+    cur = conn.cursor()
+    cur.execute("SELECT password, api_key FROM users WHERE email='{}'".format(email))
+    rows = cur.fetchall()
+
+    if len(rows) > 0:
+        if pbkdf2_sha256.verify(attempt, rows[0][0]):
+            return json.dumps({'status': 0, 'token': rows[0][1]})
```

Comment:
I suggest one of the following changes:
1.  make `status` a string, which shows the meaning of the status.
2.  add a comment to show what does `0` and `1` mean. 
1 is preferred since `status` is used all over places.